metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/LogConstants.java

```diff
@@ -13,95 +13,109 @@
 
 package com.netflix.metacat.common.server.monitoring;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+//CHECKSTYLE:OFF
 /**
  * Log constants.
  */
 public enum LogConstants {
     /**
      * General logging constants.
      */
-    GlobalPrefix("dse"), AppPrefix(GlobalPrefix + ".metacat"),
+    AppPrefix("metacat"),
+
+    /**
+     * evnets.
+     */
+    CounterEventAsync(AppPrefix + ".events.count.Async"),
```

Comment:
I think it's ok to let them be a separated counter for consistency. There're other cases like this, e.g. tableCreate, tableDelete, etc. A couple of cased the tags used are (1) the status of operations, failure/succeed; (2) the catalog/database/table names in case of the same operation.